gateway_to_backend/protocol_buffers_files/otap_message.proto

```diff
@@ -18,12 +18,49 @@ enum ScratchpadStatus {
     ERROR = 3;
 }
 
+enum ScratchpadAction {
+    // Will be the default value in case it is later extended
+    UNKNOWN_ACTION = 0;
+    // No propagation, no processing of scratchpad
+    NO_OTAP = 1;
+    // Propagate the target scratchpad but no processing
+    PROPAGATE_ONLY = 2;
+    // Propagate the target scratchpad and process it when available
+    PROPAGATE_AND_PROCESS = 3;
+    // Propagate the target scratchpad and process it after delay (starting when scratchpad is present and this info received)
+    PROPAGATE_AND_PROCESS_WITH_DELAY = 4;
+    // Otap propagation works as before with sequence comparison and processing through remote API
+    LEGACY_OTAP = 5;
+}
+
+enum ProcessingDelay {
+    UNKNOWN_DELAY = 0;
+    TEN_MINUTES = 1;
+    THIRTY_MINUTES = 2;
+    ONE_HOUR = 3;
+    SIX_HOURS = 4;
+    ONE_DAY = 5;
+    TWO_DAYS = 6;
+    FIVE_DAYS = 7;
+}
+
+
 message ScratchpadInfo {
     required uint32 len = 1;
     required uint32 crc = 2;
     required uint32 seq = 3;
 }
 
+message TargetScratchpadAndAction {
+    required ScratchpadAction action = 1; // What is the action to perform with the current scratchpad
+    optional uint32 target_sequence = 2; // Between 0 and 255 (if missing using local scratchpad sequence if present)
```

Comment:
Between 1 and 254.
I think uploading a scratchpad 255 in a Sink won't work.
And if I am correct, the single mcu api will return an error